src/pages/Login/Login.tsx

```diff
@@ -1,63 +1,59 @@
 import React, { FC, useCallback } from 'react';
-import { Link } from 'react-router-dom';
 import { useStyles } from '@grafana/ui';
-import { LoaderButton } from '@percona/platform-core';
-import { useDispatch } from 'react-redux';
-import { PublicLayout } from 'components';
-import { Routes } from 'core/routes';
-import { loginAction } from 'store/auth';
-import { Messages } from './Login.messages';
+import { OktaSignInWidget } from 'components/OktaSignInWidget';
+import { authConfig } from 'core';
+import { useOktaAuth } from '@okta/okta-react';
+import logoImg from 'assets/percona-platform-logo.svg';
+import { logger } from '@percona/platform-core';
+import bulletPoint from 'assets/bullet-point-arrow.svg';
 import { getStyles } from './Login.styles';
+import { Messages } from './Login.messages';
 
 export const LoginPage: FC = () => {
   const styles = useStyles(getStyles);
-  const dispatch = useDispatch();
+  const { oktaAuth, authState } = useOktaAuth();
+
+  const onSuccess = useCallback(
+    (tokens) => {
```

Comment:
```suggestion
  const onSuccess = useCallback((tokens) => {
```

this looks more consistent to what you did with `onError` and what we're doing elsewhere

Reply:
Oh the prettier is not allowing me to have it this way unfortunatelly